### ISSUE TITLE
new-msc-design: window draggable area is too small

### ISSUE THREAD

**Issue by gregneagle** [munki/munki#1017]
With the full window content view, you cannot move/drag the window by the usual title bar area _except_ in the left side bar. Compare with other visually-similar apps (like App Store) where you can drag using the entire width of the window and even a larger vertical area in the sidebar.

**Comment by gregneagle** [comment]
Interestingly, on Big Sur (Beta 5) you can move the window by dragging anywhere in the (former) title bar area.

But there's a weird visual glitch in that all content seems to be pushed down about 50 pixels.

![Image 9-4-20 at 8 07 AM](https://user-images.githubusercontent.com/700560/92254397-b00e2700-ee85-11ea-9ea2-9ed37ccbadfa.jpg)

**Comment by gregneagle** [comment]
I worked on this today. I overrode the mouseDown function in  MSCWebView to implement window dragging when a mouse drag happens in the normal titlebar area.
That mostly worked, though there was an area in the center of the titlebar area that was not getting the mouse down event and so drags still didn't work. This seemed to be to be about the area "Managed Software Center" took up when the window title was visible, so I tried setting the window title to an empty string. That makes the behavior even better, but there's _still_ a very small area in the middle where drags are ignored.

I can't help but feel there's some better, simpler solution to this -- I think we're supposed to get the normal dragging behavior for "free" without all this extra work, and something about how the XIB it set up is causing that not to happen.
